xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -557,6 +560,10 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
                 profile = CJNIMediaCodecInfoCodecProfileLevel::DolbyVisionProfileDvheDth;
                 break;
               case 7:
+                // set profile 8 when converting
+                if (convertDovi && CJNIBase::GetSDKVersion() >= 27)
+                  profile = CJNIMediaCodecInfoCodecProfileLevel::DolbyVisionProfileDvheSt;
+
```

Comment:
It would be good to add a code comment briefly explaining the reason why it is omitted set the profile 7 when is not converted.